test_suite/run_pypit_tests.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python
-#
-'''
+"""
 This needs to be run in the TEST_SUITES directory.
 Usually found in a Dropbox.
-'''
+
+IMPORTANT: Output will be written to TST_PYPIT env variable
```

Comment:
These are described in the README.

I wish to use TST_PYPIT as that should usually point to a folder outside the
Dropbox (to avoid those of us running tests from clobbering each other's Dropbox).
We can eliminate that once we take the TEST_SUITE out of Dropbox.